pkg/qliksense/about.go

```diff
@@ -194,8 +194,10 @@ func getImageList(yamlContent []byte) ([]string, error) {
 		})
 	}
 	var sortedImageList []string
-	for image, _ := range imageMap {
+	for image, v := range imageMap {
```

Comment:
I think that warning suggests that the range expression should be written like this (we can skip mentioning value altogether):
```
for image := range imageMap {
  sortedImageList = append(sortedImageList, image)
}
```